### ISSUE TITLE
Raw CSS @imports no longer work

### ISSUE THREAD

**Issue by samuelcolvin** [sass/libsass-python#245]
See https://github.com/sass/libsass/releases/tag/3.5.3

1. ~~This changed the current behaviour of libass-python such that importing raw css files without the `.css` extension no longer works. Was this intentional? Might have been useful to include this in the changelog.~~ see below
2. ~~I'm not seeing the deprecation warnings noted above. Maybe I'm not logging warnings or something?~~ see below
3. I want to continue to be able to import raw `css` files after 3.6 is released, will this require any changes to libsass-python? Eg. to support `sass_option_push_import_extension`

**Comment by samuelcolvin** [comment]
Sorry, I was being dumb above.

Importing css files is not working since the release of v14.3. As always you can do a css "import" eg. reference a separate file by using `@import 'foobar.css';` but that's not what I'm looking for. I want `@import 'foobar';` to truly import `foobar.css` into the compiles sass.

I guess this means libass-python need to support `sass_option_push_import_extension`, perhaps it already does and I need to work out how to use it, looking into it now

Part of the problem is that the 3.5.3 release notes are confusing

> An `@import` that resolves to a `.css` file will produce a deprecation
warning. This feature will be removed in the upcoming 3.6 release.

For me at least it's just downright failing rather than giving a deprecation
warning.

**Comment by asottile** [comment]
CC @xzyfer -- this seems to be a regression from `libsass`'s side:

```
$ tail -n999 test.scss wat.css 
==> test.scss <==
@import 'wat';

==> wat.css <==
a {
    b: c;
}
$ pysassc --version
pysassc 0.14.2 (sass/libsass 3.5.2)
$ pysassc test.scss 
a {
  b: c; }
$ pip install --upgrade libsass
...
$ pysassc --version
pysassc 0.14.3 (sass/libsass 3.5.3)
$ pysassc test.scss 
pysassc: error: Error: File to import not found or unreadable: wat.
        on line 1 of test.scss
>> @import 'wat';

   ^
```

**Comment by xzyfer** [comment]
Apologies this breaking change unintentionally landed in 3.5.3 instead of 3.6.0.

We plan to roll forward with the breaking change, and have update the release notes accordingly.
